dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/trigramsummary/TrigramSummaryControllerTest.java

```diff
@@ -185,70 +197,51 @@ void setUp()
     void getTrigramIndexSummaryWhenNoIndexesAreCreated()
     {
 
-        when( trackedEntityAttributeTableManager.getAttributeIdsWithTrigramIndexCreated() )
+        when( trackedEntityAttributeTableManager.getAttributeIdsWithTrigramIndex() )
             .thenReturn( new ArrayList<>() );
+
         TrigramSummary trigramSummary = controller.getTrigramSummary( new HashMap<>() );
 
         assertNotNull( trigramSummary );
-
-        Set<String> expected = Set.of( "\"AttributeA\"", "\"AttributeB\"", "\"AttributeF\"" );
-        assertAttributeList( trigramSummary.getIndexableAttributes(), expected );
-
-        Set<String> expected2 = Set.of();
-        assertAttributeList( trigramSummary.getIndexedAttributes(), expected2 );
-
-        Set<String> expected3 = Set.of();
-        assertAttributeList( trigramSummary.getObsoleteIndexedAttributes(), expected3 );
-
+        assertAttributeList( trigramSummary.getIndexableAttributes(),
+            Set.of( "\"AttributeA\"", "\"AttributeB\"", "\"AttributeF\"" ) );
+        assertAttributeList( trigramSummary.getIndexedAttributes(), Set.of() );
```

Comment:
I would use Collections.emptySet() instead of Set.of()
but that is a nitpick (in a test) 😋